h/routes.py

```diff
@@ -305,4 +305,5 @@ def includeme(config):  # noqa: PLR0915
 
     # ORCID
     config.add_route("oidc.connect.orcid", "/oidc/connect/orcid")
+    config.add_route("oidc.login.orcid", "/oidc/login/orcid")
```

Comment:
The new route that the _Log in with ORCID_ button on the login and signup pages will link to.